src/category_theory/triangulated/pretriangulated.lean

```diff
@@ -144,16 +140,6 @@ by simpa using comp_dist_triangle_mor_zero₁₂ C (T.rotate.rotate) H₂
 TODO: If `C` is pretriangulated with respect to a shift,
 then `Cᵒᵖ` is pretriangulated with respect to the inverse shift.
 -/
-end pretriangulated
-end category_theory.triangulated
-
-namespace category_theory.triangulated
-namespace pretriangulated
-
-variables (C : Type u₁) [category.{v₁} C] [has_zero_object C] [has_shift C ℤ] [preadditive C]
-  [∀ n : ℤ, functor.additive (shift_functor C n)]
-variables (D : Type u₂) [category.{v₂} D] [has_zero_object D] [has_shift D ℤ] [preadditive D]
-  [∀ n : ℤ, functor.additive (shift_functor D n)]
```

Comment:
Some stuff below (`triangulated_functor_struct`) previously does not require a pretriangulated instance. Is this a conscious change? Will we ever need to consider such functors before we have a pretriangulated instance?

Reply:
Thanks. It was not a conscious choice. I have just reverted this. (Anyway, I plan to refactor the notion of functor which commutes with shift, by introducing shift isomorphisms for all integers [with some compatibilities] instead of the shift by `1` only.)